<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hoang-Tuan" failures="1" tests="2" name="Suit test for running test cases" time="5.415" errors="0" timestamp="2022-02-07T15:18:58 ICT">
  <testcase classname="Tiki.TikiTest" name="Testcase003" time="2.722">
    <failure type="java.lang.AssertionError" message="Verification failed.">
      <![CDATA[java.lang.AssertionError: Verification failed.
at ExtentReport.TestListener.afterInvocation(TestListener.java:146)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- Testcase003 -->
  <testcase classname="Tiki.TikiTest" name="Testcase004" time="0.002"/>
</testsuite> <!-- Suit test for running test cases -->
